apps/credential-service/lib/server.ts

```diff
@@ -21,7 +21,12 @@ interface RequestParameterHeaders extends IncomingHttpHeaders {
 
 /** Regex for matching authorization header uses. */
 const authorizationRegex = new RegExp(/Bearer (.*)/);
-/** The length of time in ms between heartbeat checks. */
+/**
+ * The length of time in ms between heartbeat checks.
+ *
+ * 10 seconds is used as during the local PR validation process, the javascript thread
+ * becomes blocked, and can cause a timeout.
+ */
```

Comment:
Ohh, that is pretty interesting. Couldn't we offload the heartbeat somehow? It seems pretty brittle that we need a large enough timeout here (especially if we add stuff in the future / use it for other tools).

Reply:
Yeah, I think it makes sense in a follow up to do once we have everything in place as part of larger testing.

Reply:
Sounds good